src/components/hero/hero.scss

```diff
@@ -0,0 +1,109 @@
+@font-face {
+    font-family: "Josefin";
+    src: url("../font/JosefinSans-VariableFont_wght.ttf");
+}
+@font-face {
+    font-family: Lato;
+    src: url("../font/Lato-Black.ttf");
+}
+$font-family:Josefin;
+$font-family2:Lato;
+
+.hero{
+    background-color: #F2F0FF;
```

Comment:
please create "_variables.scss" file if you don't have it yet, and declare variables for colors there
after that you've got to use those variables in this file